### DISCUSSION TITLE
Can not generate QR Code if characters more than 154

### DISCUSSION THREAD

**Discussion by uuProxy** [chillerlan/php-qrcode#140] [Q&A] [answered]
For example, characters in render counts 155, failed:

$qrcode = (new QRCode($options))->render('sFQlFP2yIJlY4zcvFtDrA8GNTEkVosaQhfkPlEMbf9ZHaE8YYSFFPH94WOxWKk70Jtydwf7FgD257hXnZWoNoWb0V9z444kQq3Fx8SM38IHSGDMytbebAu7SP1F2AxvyIlFEQY0eUwIm7cb9KGql8b4TXeZ');

But remove one character it generates successfully.

**Comment by codemasher** [comment, marked as answer]
Looks like your `$options->version` setting is too low for the length of the data string., Try setting it to a higher number or `QRCode::VERSION_AUTO` instead. See also https://www.qrcode.com/en/about/version.html

**Reply by uuProxy** [reply]
Thank you, it works, I set the version above 7  if the characters are 155, and 10, if the characters are more than 250.
